README.md

```diff
@@ -1 +1,33 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Biography
+
+My name is Mads A. Hoofnagle. I am 24 years old and I live in Chicago, Illinois. I am in the Masters of Arts Program in Social Sciences at the University of Chicago. I have lived in *four different countries*: The U.S., Ireland, Iceland, and Denmark. I speak English and French fluently and some Icelandic, as well. 
+
+## Education
+
+I have **two undergraduate degrees**: a B.A. in psychology and a B.S. in anthropology from the University of Iowa. I have a M.A. in Viking and Medieval Norse Studies from the University of Iceland. I also have a certificate in Irish Studies from the University College Cork in Ireland. 
+
+## Academic Interests
+
+I used to be focused primarily on archaeology, but have since moved away from that in my current education to study medical anthropology. I am writing my current M.A. thesis on cultural facets of depression in Iceland. 
+
+## Hobbies and Extracurriculars
+
+I don't have much time right now to pursue my hobbies and interests outside of school, but I play a lot of video games. Currently, I've been spending a lot of time on: 
+
+1. Darkest Dungeon
+2. Dead by Daylight
+3. DOTA 2
+4. Call of Duty: Modern Warfare
+
+I also play in a Dungeons and Dragons campaign with some of my friends who live all around the world. We are currently hunting vampires! 
+
+When it comes to staying in shape, I like to go for a run or do some yoga. 
+
+Here is a picture of me playing DOTA 2 and having a beer:
+
+![Support Main](https://media.discordapp.net/attachments/400011446068576256/699409356294324254/IMG_2519.jpg?width=508&height=677)
+
+And [here](https://www.youtube.com/watch?v=ox_-BOCLv8s) is a link to an animation that was drawn to accompany a clip from one of my favorite podcasts, My Brother My Brother and Me. 
+
```

Comment:
The placement of these lines makes it a little hard to see. I would suggest placing this somewhere before the image or making the image smaller